changelog.d/20240327_224004_tahamukhtar20.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Fixed typescript linting errors in ts files
```

Comment:
This fix has no impact on the user, so I don't think it needs a changelog entry.